apps/src/templates/sectionsRefresh/QuickAssignTableHelpers.jsx

```diff
@@ -96,6 +96,14 @@ function updateSectionCourse(updateCourse, course) {
     )?.id;
   }
 
+  // If no recommended version, fall back to the most recent stable version.
+  if (courseVersionId === undefined) {
+    const stableVersions = Object.values(courseVersions).filter(
+      version => version.is_stable
+    );
+    courseVersionId = stableVersions[stableVersions.length - 1]?.id;
```

Comment:
Do we know  for sure that these will always be in chronological order?

Reply:
ah, no, that was just an assumption I made 😅 I'll check the api, and see if I need to sort these by something instead.

Reply:
it looks like the `key` is consistently a "version year", so I'm sorting by that instead. unless you know of any exceptions to that rule?